content/rest-api/beta.textile

```diff
@@ -30,6 +30,8 @@ This means that caller can usually make the call and have access to the success
 
 All batch-mode api calls are limited to addressing at most 100 different channels per request.
 
+All POST/PUT requests (such as bulk publish) are subject to a maximum request body size of 2MB.
```

Comment:
I think @POST@ and @PUT@ should be styled as code blocks IMHO

Reply:
We use `MiB` style for byte sizes unilaterally to avoid confusion in some instances where `MB` can mean 2000 KiB or can also mean 2048 KiB.  We had a big discussion on this before, but sadly I can't find a link to it.

Reply:
I found what we documented, if you look in our doc ["Ably Realtime Copy Guidelines & Dictionary"](https://docs.google.com/document/d/1elqHBcMoNk3cOkxcJtd-t4v8WW55d3EfBVRYvT88mvQ/edit), there is a section:

Bandwidth and storage uses the [IEC standard](https://en.wikipedia.org/wiki/IEC_60027) always to avoid ambiguity.  Therefore, we use bytes, KiB, MiB, GiB, and TiB.  We never use KB, GB, TB as one KB can mean 1,000 bytes or 1,024 bytes.  See [internal discussion](https://github.com/ably/website/issues/1597).